### PR TITLE
Add recipe for pyupgrade [skip appveyor] [skip travis]

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

File: recipes/pyupgrade/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  home: https://github.com/asottile/pyupgrade
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
+  summary: A tool to automatically upgrade syntax for newer versions.
+
+extra:
+  recipe-maintainers:
+    - nicoddemus
+    - tadeu
```
Comment (tadeu) [inline recipes/pyupgrade/meta.yaml]: 👍

**Comment by nicoddemus** [comment]
@conda-forge/help-python @igortg @ocefpaf ready for review 👍

File: recipes/pyupgrade/LICENSE [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2017 Anthony Sottile
```
Comment (beckermr) [inline recipes/pyupgrade/LICENSE]: Let's make sure this is submitted upstream.

Comment (nicoddemus) [reply recipes/pyupgrade/LICENSE]: Done: https://github.com/asottile/pyupgrade/pull/90

File: recipes/pyupgrade/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+test:
+  commands:
+    - pyupgrade --help
+
+about:
+  home: https://github.com/asottile/pyupgrade
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (ocefpaf) [inline recipes/pyupgrade/meta.yaml]: `{{ environ["RECIPE_DIR"] }}/LICENSE`

I'm not 100% sure but I believe you can do just `LICENSE` here, let conda find it on the recipe or the source, and be "future proof" when the license is available in the source and removed from the recipe.

Comment (nicoddemus) [reply recipes/pyupgrade/meta.yaml]: Yeah it worked for me locally, thanks, didn't know that trick yet. 👍

**Pull request merged**